packages/api/src/models/Validator.js

```diff
@@ -1,12 +1,46 @@
+const BlockHeader = require('./BlockHeader')
+
 module.exports = class Validator {
   proTxHash
+  proposedBlocksAmount
+  lastProposedBlockHeader
 
-  constructor (proTxHash) {
+  constructor (
+    proTxHash,
+    proposedBlocksAmount,
+    lastProposedBlockHeader
+  ) {
     this.proTxHash = proTxHash
+    this.proposedBlocksAmount = Number(proposedBlocksAmount)
+    this.lastProposedBlockHeader = lastProposedBlockHeader
```

Comment:
We should default to null in this place like in other models

Reply:
✅